src/RoboSaverVirtualModule.sol

```diff
@@ -26,6 +26,7 @@ contract RoboSaverVirtualModule {
     enum PoolAction {
         WITHDRAW,
         DEPOSIT,
+        CLOSE,
```

Comment:
missing here natspec comment for the new value `CLOSE`

Reply:
69012ed